docs/getting_started/installation_guide/platforms/aws-image.rst

```diff
@@ -0,0 +1,35 @@
+.. _aws-image:
+
+Amazon Machine Images
+=====================
+
+Hopsworks 1.4.1 is available as a community AMI ready to be launched. You can use it to
```

Comment:
No need to mention the version here as the docs are already versioned and we will forget to update with subsequent releases